components/suggestion/suggestion_list.jsx

```diff
@@ -272,12 +276,17 @@ export default class SuggestionList extends React.PureComponent {
         }
         const mainClass = 'suggestion-list suggestion-list--' + this.props.position;
         const contentClass = 'suggestion-list__content suggestion-list__content--' + this.props.position;
-
         return (
             <div
                 ref={this.wrapperRef}
                 className={mainClass}
             >
+                {this.state.component === 'EmoticonSuggestion' &&
+                <div
+                    className='sr-only'
+                    aria-label={'emojy ' + this.state.focused}
+                    aria-live='polite'
+                >{'emojy ' + this.state.focused}</div>}
```

Comment:
A few things here:
1. I like deriving this from `this.props.selection`, but let's not take off the leading colon with a `substring` like you were doing so that this supports all types of autocomplete suggestions.
2. This should use a `FormattedMessage` component so that the text can be translated, and we shouldn't just concatenate on the selected value because some languages might order these words differently. We can replace this with something like

    ```js
    <div aria-live='polite'>
        <FormattedMessage
            id='suggestion.selectedReadout'
            defaultMessage='suggestion {value} selected'
            values={{this.props.selection}}
        />
    </div>
    ```
    After adding that, you'll also have to run `make i18n-extract` so that the translation files are correctly updated. I also asked what this text should be exactly
4. Minor thing, but `emojy` should be spelled `emoji`

Reply:
turns out, all of this was'nt needed. just a check in generateLabel(). That's all.

Reply:
Thankyou for the suggestions though

Reply:
Huh, that's convenient. I didn't realize we already had a lot of the stuff in place for that. That's good to know